CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [13.5.1] - 2023-04-26
```

Comment:
we need to update the minor version here and not the patch version since it's a not really a bug fix.